manim/mobject/mobject.py

```diff
@@ -2215,14 +2250,16 @@ def get_nadir(self) -> Point3D:
 
     def length_over_dim(self, dim: int) -> float:
         """Measure the length of an :class:`~.Mobject` in a certain direction."""
-        return self.reduce_across_dimension(
+        max_distance: float = self.reduce_across_dimension(
             max,
             dim,
-        ) - self.reduce_across_dimension(min, dim)
+        )
+        min_distance: float = self.reduce_across_dimension(min, dim)
+        return max_distance - min_distance
```

Comment:
`max_distance` and `min_distance` are a bit misleading, because distances are always positive or zero. Here, the values returned by `reduce_across_dimension()` can be negative and are more related to coordinates in space, so a better option could be `max_coord` and `min_coord`. For example, for a `Mobject` which spans from x = -3 to x = 5, `max_coord` would be the max X coordinate 5 and `min_coord` would be the min X coordinate -3, so the length over the X dimension would be 8.